### DISCUSSION TITLE
[Source|Target]EncryptConnection parameter values?

### DISCUSSION THREAD

**Discussion by FH-Inway** [microsoft/DacFx#185] [Q&A] [answered]
In wake of the [encrypt connection breaking changes](https://aka.ms/dacfx-connection), I'm wondering about the available parameter values for the `[Source|Target]EncryptConnection` parameters. `True` and `False` seem to be clear, but I'm less sure about `Optional`, `Strict` and `Mandatory`. 
`Optional` currently gives me the same result as `False`, while `Mandatory` gives the same result as `True`.
For `Strict`, the export action gets stuck on the "Connecting to database 'mydb' on server 'myserver'." message.

Is there more information available about the behavior of these parameter values?

**Comment by ErikEJ** [comment, marked as answer]
There is some detailed info here: https://learn.microsoft.com/en-us/sql/relational-databases/security/networking/tds-8-and-tls-1-3?view=sql-server-ver16
